srcpkgs/systemd-boot/template

```diff
@@ -28,16 +29,20 @@ configure_args="-Defi=true -Dman=enabled -Dbootloader=enabled
 # most of these aren't needed for what we're building but it's easier than patching
 hostmakedepends="pkg-config gperf python3-Jinja2 python3-pyelftools libxslt docbook-xsl"
 makedepends="libcap-devel libmount-devel"
-short_desc="UEFI stub loader from systemd-boot"
-maintainer="classabbyamp <void@placeviolette.net>"
+depends="systemd-boot-efistub>=${version}_${revision} ukify>=${version}_${revision}"
+short_desc="Simple UEFI boot manager from systemd"
+maintainer="Daniel Martinez <danielmartinez@cock.li>"
 license="LGPL-2.1-or-later"
 homepage="https://www.freedesktop.org/wiki/Software/systemd/systemd-boot/"
-distfiles="https://github.com/systemd/systemd-stable/archive/refs/tags/v${version}.tar.gz"
-checksum=ca40487e044c9d6b7ad214d453b1cab696e5ccbd5244eca78460972cb0589a17
-make_check=no  # missing printf.h
+distfiles="https://github.com/systemd/systemd/archive/refs/tags/v${version}.tar.gz"
```

Comment:
changed from `systemd-stable` to `systemd`, since the stable repo seems to be for backports to previous major releases. There are several minor releases since version 256 which are not in `systemd-stable`